src/popper.ts

```diff
@@ -79,13 +85,13 @@ export function createArrowElement(
 ): HTMLDivElement {
   const arrow = div()
   if (arrowType === 'round') {
-    arrow.className = 'tippy-roundarrow'
+    arrow.className = selectorToClassName(ROUND_ARROW_SELECTOR)
```

Comment:
instead of introducing a new function here, maybe we could create another constant for this?
e.g., we could have `ROUND_ARROW_CLASS` *and* `ROUND_ARROW_SELECTOR`

Reply:
Was trying to save as many bytes as possible